src/applications/vaos/containers/AppointmentsPage.jsx

```diff
@@ -54,13 +78,55 @@ export class AppointmentsPage extends Component {
     }
   }
 
+  onSelectTab = tabIndex => {
+    const { futureStatus, pastStatus } = this.props.appointments;
+    this.setState({ tabIndex });
+
+    let path = '/health-care/schedule-view-va-appointments/appointments/';
+
+    if (tabIndex === 0) {
+      if (futureStatus === FETCH_STATUS.notStarted) {
+        this.props.fetchFutureAppointments();
+      }
+    } else if (tabIndex === 1) {
+      if (pastStatus === FETCH_STATUS.notStarted) {
+        this.fetchPastAppointments();
+      }
+
+      path = appendQuery(path, { view: 'past' });
+    }
+
+    browserHistory.push(path);
```

Comment:
This will update the url, but I don't think the UI will change if you use the browser back and forward buttons?

I think ultimately we'd want to take advantage of react router for this, where we could have AppointmentsPage and FutureAppointments and PastAppointments sub pages.

Reply:
Right, do you happen to have an example of us doing this elsewhere in `vets-website`?

Reply:
Claim status does. The main claim page has several sub pages.

Reply:
We don't have to do that in this PR, though.

Reply:
Awesome, thanks.  Created a new ticket to handle browser back/forward: https://github.com/department-of-veterans-affairs/va.gov-team/issues/7427